sfe/templates/layout.html

```diff
@@ -0,0 +1,17 @@
+{{define "meta"}}
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    
+        <link href="https://fonts.googleapis.com/css?family=Nunito:300,400,600,700,800" rel="stylesheet">
```

Comment:
I'd prefer not to pull in fonts from third-party URLs; let's just drop Nunito and use a default font.

Reply:
@breakdancingcat sent me [this](https://github.com/system-fonts/modern-font-stacks) and I'm storing it here for posterity.